docs/reference/environment.md

```diff
@@ -0,0 +1,42 @@
+# Environment Variables
+
+Screwdriver exports a set of environment variables that you can rely on during
+the course of a build. These are:
+
+## Continuous Integration
+| Name | Description |
+|------|-------------|
+| SCREWDRIVER | Set to true if we are running in Screwdriver |
+| CI | Set to true if we are running in Screwdriver |
```

Comment:
This is actually used in Travis and Wercker https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables